backend/src/main/java/de/kathahuth/gymast/model/Group.java

```diff
@@ -9,9 +10,12 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
+@Builder
```

Comment:
please keep in mind that `@builder` enables you to build incomplete objects which may cause problems if you rely on complete objects in your data base